spec/abci++/abci++_app_requirements_002_draft.md

```diff
@@ -672,6 +675,21 @@ node has received all precommits for a block, forgoing the remaining commit time
 Setting this parameter to `false` (the default) causes Tendermint to wait
 for the full commit timeout configured in `TimeoutParams.Commit`.
 
+##### ABCIParams.VoteExtensionsEnableHeight
+
+This configures the height at which vote extensions become mandatory.
+The default value is 0, and means that so far the Application has not
+set the height yet.
+When the configured height `H` is reached, `PrepareProposal` will not
+include vote extensions yet, but `ExtendVote` and `VerifyVoteExtension` will
+be called. Then, when reaching height `H+1`, `PrepareProposal` will
+include the vote extensions from height `H`. From this point on, vote extensions
+are mandatory and cannot be disabled, although the application can provide 0-length
```

Comment:
Should we define what mandatory means? i.e. that chains will not make progress if votes don't include extensions.